<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntermodClass</class>
 <widget class="QMainWindow" name="IntermodClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RF Intermodulation Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget{
	background-color: #5a5c5e;
}

QLabel{
	Font: 14pt &quot;Arial&quot;;
	color: white;
}

QPushButton{
	font: 14pt &quot;Arial&quot;;
	color: white;
	background-color: #282c30;
	border-radius: 5px;
}
QPushButton::pressed{
	background-color: #577590; 
}
QPushButton:hover:!pressed{
	background-color: #739bbe; 
}

QLineEdit{
	font: 14pt &quot;Arial&quot;;
	background-color: #282c30;
	border-radius: 5px;
	color: white;
	padding-left: 10;
}

QComboBox{
	font: 14pt &quot;Arial&quot;;
	color: white;
	background-color: #282c30;
	border-radius: 5px;
	padding-left: 30px;
}

QComboBox::drop-down{
	width: 25;
}

QMessageBox {
	background-color: #0D1218;
	Font: 14pt &quot;Arial&quot;;
}
QMessageBox QPushButton {
	font: 14pt &quot;Arial&quot;;
	color: white;
	background-color: #1e2733    ;
	border-radius: 5px;
	height: 40;
	width: 100;
}

/*TABLE VIEW */
QTableView{
	border-radius: 8;
	background-color: #3f464d   ;
	Font: 14pt;
	color: white;
	gridline-color: #1C2321;
}

QTableView:item:selected:focus {background-color: #577590 ;}
QTableView:item:selected {background-color: #577590 ;}

QTableView QTableCornerButton::section {
	border-top-left-radius: 8;
	background-color: #5F6366;
}

QHeaderView::section {
	background-color: #282c30   ;
	color: white;
	font: Bold 14pt;
	padding: 5;
	padding-left: 4px;
	border: 1px solid #6c6c6c;
}
QHeaderView::section:checked
{
	background-color:#0D1218   ;
}

/*SCROLL BAR*/
QScrollBar:vertical {
     background:white;
     width: 10;
     margin: 0px 0 0px 0;
	 border-radius: 2;
 }
 QScrollBar::handle:vertical {
     background: #0D1218        ;
     min-height: 30px;
	border-radius: 2;
 }
QScrollBar::add-line:vertical {
     background: #0D1218  ;
     height: 0px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     background: #0D1218  ;
     height: 0px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2" colspan="5">
       <widget class="QFrame" name="TitleBanner">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color:#282c30;
	border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="logoLabel">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="styleSheet">
            <string notr="true">Font: Bold 18pt;</string>
           </property>
           <property name="text">
            <string>RCMP E-Division RTP Intermodulation Calculator</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="minButton">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Minimize</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton::pressed{
	background-color: #577590; 
}
QPushButton:hover:!pressed
{
 background-color:#739bbe;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maxButton">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Maximize</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton::pressed{
	background-color: #577590; 
}
QPushButton:hover:!pressed
{
 background-color:#739bbe;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Exit</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton::pressed{
	background-color: #577590; 
}
QPushButton:hover:!pressed
{
 background-color:#739bbe;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <layout class="QHBoxLayout" name="titleLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titleLine">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Save Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Open Project</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="4">
       <widget class="QPushButton" name="calculateButton">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Run the intermodulation calculation</string>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="helpButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Help dialog</string>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>350</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Enter frequencies for the intermodulation calculation</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>225</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Transmit Freq (MHz)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Recieve Freq (MHz)</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="4">
         <widget class="QLabel" name="toleranceLabel">
          <property name="text">
           <string>+/- kHz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="cutoffLabel">
          <property name="text">
           <string>Calculation Cutoff:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="searchLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Search window for hits</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="searchWindowLabel">
          <property name="text">
           <string>Search Window:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QComboBox" name="orderBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Calculation order of results</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>3rd Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19th Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21st Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23rd Order</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25th Order</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="8">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="aboutButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="clearButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear table of input values</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save       Ctrl-S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open     Ctrl-O</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print       Ctrl-P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IntermodGUI.qrc"/>
 </resources>
 <connections/>
</ui>
